### PR TITLE
feat: merge the studio next and studio apps

### PR DESCRIPTION
**Pull request by catosaurusrex2003**
**Description**

In the first commit, I made necessary changes to the package.json files located both in the root directory and within the studio-next folder. (Please refer to this specific commit to review the changes, as in the subsequent commit I deleted the entire studio folder along with its files.)

After these changes, I thoroughly checked for any regressions or differences between the studio-next app and the currently deployed version of the site.
I did not find any discrepancies or issues during this process.

**Resolves Issue**: #1134 

**Some questions i have**
1. Should we continue versioning from the last version of `studio-next`, or should we maintain the versioning from the old `studio` folder?

This is my first time performing a migration within a monorepo structure, and I would appreciate any guidance or feedback on best practices and to ensure the migration process follows the project's conventions.

### CODE REVIEW

**Comment by Shurtu-gal** [comment]
@catosaurusrex2003 could you resolve the conflicts.

**Comment by aeworxet** [comment]
https://github.com/asyncapi/studio/pull/1120 was merged just now.

**Comment by catosaurusrex2003** [comment]
very beautiful PR 🤩
Resolving conflicts asap.

**Comment by Shurtu-gal** [comment]
> very beautiful PR 🤩 Resolved conflicts. Now onto migrating content of `studio-next` into `studio`

Great. 

Also I guess @fmvilas or @derberg someone among you would have to do some changes on netlify side before we merge this.
Basically need to change the runtime to `Next.js`.

File: .sonarcloud.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Disable specific file since it would introduce more complexity to reduce it - mainly code complexity and complex template literals
-sonar.exclusions=apps/studio/public/js/monaco/**,apps/studio/src/tailwind.css,apps/studio/src/components/SplitPane/**,apps/studio-next/cypress/**,apps/studio-next/Dockerfile
+sonar.exclusions=apps/studio/public/js/monaco/**,apps/studio/src/tailwind.css,apps/studio/src/components/SplitPane/**,apps/studio/cypress/**,apps/studio/Dockerfile
 # Disable duplicate code in tests since it would introduce more complexity to reduce it.
-sonar.cpd.exclusions=apps/studio/**,apps/studio-next/src/components/Navigationv3.tsx,apps/studio-next/src/components/Navigation.tsx,apps/studio-next/cypress/**,apps/studio-next/src/helpers/driver.ts
+sonar.cpd.exclusions=apps/studio/**,apps/studio/src/components/Navigationv3.tsx,apps/studio/src/components/Navigation.tsx,apps/studio/cypress/**,apps/studio/src/helpers/driver.ts
```
Comment (Shurtu-gal) [inline .sonarcloud.properties:4]: Remove the cpd exclusions which were not there before migration.

File: apps/studio/CHANGELOG.md
Comment (Shurtu-gal) [inline apps/studio/CHANGELOG.md:1]: Leave the `CHANGELOG` as it is.

File: apps/studio/.eslintignore
Comment (Shurtu-gal) [inline apps/studio/.eslintignore:1]: Any reason for its deletion?

File: apps/studio/Dockerfile
Comment (Shurtu-gal) [inline apps/studio/Dockerfile:1]: Same for this.

File: apps/studio/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/studio",
-  "version": "0.21.4",
+  "version": "0.1.4",
```
Comment (Shurtu-gal) [inline apps/studio/package.json:3]: Let the version be the same.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: .sonarcloud.properties (hunk 1 above)
Comment (catosaurusrex2003) [reply .sonarcloud.properties:4]: ah yes, didnt realise the changes i made are redundant.

File: apps/studio/.eslintignore
Comment (catosaurusrex2003) [reply apps/studio/.eslintignore:1]: `studio-next` didnt require a .eslintignore so i figured it wouldnt require after migration also.
I will add it back.

File: apps/studio/Dockerfile
Comment (catosaurusrex2003) [reply apps/studio/Dockerfile:1]: lost track of this file and accidentally deleted.
adding it back.

**Comment by catosaurusrex2003** [comment]
@Shurtu-gal 
`studio` is getting built and deployed correctly but not reachable on the preview link.
I think this requires neccesary changes from netlify side.

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal `studio` is getting built and deployed correctly but not reachable on the preview link. I think this requires neccesary changes from netlify side.

Yeah, I have tagged @fmvilas in a previous comment regarding this.

**Comment by fmvilas** [comment]
Thanks for the ping, @Shurtu-gal. It's working now 👍

**Comment by Shurtu-gal** [comment]
@catosaurusrex2003 please check the failing tests

**Comment by catosaurusrex2003** [comment]
@Shurtu-gal i fixed the ones failing due to linting

The ones failing now are due to a missing `studio-next` folder.

**Comment by aeworxet** [comment]
@Shurtu-gal
I have a feeling that Sonar issues go beyond the initial scope of this issue and 'good first issue' level. Should they be handled by you (by fixing and pushing to the @catosaurusrex2003's branch)?

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal I have a feeling that Sonar issues go beyond the initial scope of this issue and 'good first issue' level. Should they be handled by you (by fixing and pushing to the @catosaurusrex2003's branch)?

Yeah, I have already talked with him and we are doing that.

Also @fmvilas I think there is some problem with the config as the share feature is working in both mine and @catosaurusrex2003 deploys.

**Comment by Shurtu-gal** [comment]
I will fix the Sonar Cloud Issues by tomorrow. Just need to correct the netlify config for functions.

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal I have a feeling that Sonar issues go beyond the initial scope of this issue and 'good first issue' level. Should they be handled by you (by fixing and pushing to the @catosaurusrex2003's branch)?

I was going through the sonarcloud issues. Most of them were already present and we don't have a trivial way of solving the `tabIndex` issue as it would require to replace `list` with `list of buttons`. 

I have fixed the multiple react problems though and added a change-set. Going ahead with the merge.

**Review by Shurtu-gal — APPROVED** [review]

**Pull request merged**